source/docs/zero-to-robot/step-3/radio-programming.rst

```diff
@@ -2,198 +2,112 @@
-  - Broadcast address ``10.TE.AM.255``
+1. Select :guilabel:`Robot Radio Mode`
 
-- DNS server enabled. DNS server IP and domain suffix (``.lan``) are served as part of the DHCP.
+2. Enter the team number
 
-At home only:
+3. Enter the suffix, if desired.  This will help identify you robot and distinguish it from other networks.
 
-- SSID may have a "Robot Name" appended to the team number to distinguish multiple networks.
-- Firewall option may be enabled to mimic the field firewall rules (open ports may be found in the Game Manual)
+4. Enter the 6 GHz WPA/SAE key.  This key will need to match the key on the Acess Point you configure.
 
-.. warning:: It is not possible to modify the configuration manually.
+5. Enter the 2.4 GHz WPA/SAE key.  This is the password team members will type in when connecting to the 2.4 GHz network, if available.
 
-## Install the Software
+## Access Point Radio Configuration
 
-.. image:: images/radio-programming/radio-installer.png
-  :alt: The radio configuration installer .exe file in windows explorer.
+Follow all of the same steps as the robot radio configuration instead choosing :guilabel:`Access Point Mode` at the top of the configuration page.
 
-Double click on ``FRC_Radio_Configuration_VERSION.exe`` to launch the installer. Follow the prompts to complete the installation.
+## Alternative Setup Discussion
 
-Part of the installation prompts will include installing Npcap if it is not already present. The Npcap installer contains a number of checkboxes to configure the install. You should leave the options as the defaults.
+### Optimal Setup: Two VH-109 Radios
 
-## Launch the software
+For the best experience and to closely simulate field conditions, it is strongly recommended that your team uses two VH-109 radios during testing and preparation. This dual-radio setup mirrors the competition environment, ensuring your robot operates under realistic network conditions. Additionally, having two radios allows you to fully leverage the high-speed, low-latency communication provided by the 6GHz band, which is crucial for optimal robot performance in high-stakes scenarios.
 
-.. image:: images/radio-programming/radio-launch.png
-  :alt: The FRC Radio Configuration Utility in the start menu.
+### Only 1 VH-109 radio
 
-Use the Start menu or desktop shortcut to launch the program.
+If your team has access to only one VH-109 radio, there are still viable options to continue testing and preparing for competition. However, these setups require additional considerations:
 
-.. note:: If you need to locate the program, it is installed to ``C:\Program Files (x86)\FRC Radio Configuration Utility``. For 32-bit machines the path is ``C:\Program Files\FRC Radio Configuration Utility``
+#### Use an Old Radio for Testing
 
-## Allow the program to make changes, if prompted
+If your team still has an older radio from a previous season, it can serve as a temporary substitute for a second VH-109. In this case, you should:
```

Comment:
"second VH-109" sounds like you would be using a VH-109 and and OM5P together. 
Maybe: "If your team still has an older radio from a previous season, it can serve as a temporary substitute for the VH-109 on your robot, allowing you to connect directly over 5GHz".